ocdaction/profiles/models.py

```diff
@@ -2,9 +2,15 @@
 
 from django.db import models
 from custom_user.models import AbstractEmailUser
+import datetime
 
 class OCDActionUser(AbstractEmailUser):
 
     # basic info
     username = models.CharField(max_length=24, blank=True)
-    
\ No newline at end of file
+    date_birth = models.DateField('date of birth', null=True, blank=True)
+
+    def is_date_of_birth_valid(self):
```

Comment:
sorry forgot to check this, this is not required here we will validate in the form instead as that is where we care about the age being real and we want to throw an error at form level. So please remove for now, database doesn't need to care about the date of birth being valid.